src/Tooltip/Tooltip.js

```diff
@@ -381,19 +377,30 @@ class Tooltip extends React.Component<AllProps, State> {
             />
           )}
         </Target>
-        <Popper placement={placement} className={classes.popper} {...PopperProps}>
-          <div
-            id={id}
-            role="tooltip"
-            aria-hidden={!open}
-            className={classNames(
-              classes.tooltip,
-              { [classes.tooltipOpen]: open },
-              classes[`tooltip${capitalizeFirstLetter(placement.split('-')[0])}`],
-            )}
-          >
-            {title}
-          </div>
+        <Popper placement={placement} {...PopperProps}>
+          {({ popperProps }) => {
+            const scale = open ? 1 : 0;
+            const popperStyle = Object.assign(popperProps.style, {
+              transform: `${popperProps.style.transform} scale(${scale})`,
```

Comment:
There is no guarantee that the transform property will be present, nor will work. I'm gonna find something else.
For reference. https://github.com/FezVrasta/popper.js/blob/46ad4238fcd91ccb7b4f16e68a425d86e2098d1d/packages/popper/src/utils/getSupportedPropertyName.js